emcc.py

```diff
@@ -394,18 +394,32 @@ def find_output_arg(args):
   """
   outargs = []
   specified_target = None
-  use_next = False
-  for arg in args:
-    if use_next:
-      specified_target = arg
-      use_next = False
-      continue
+
+  arg_count = len(args)
+  arg_index = 0
+
+  while arg_index < arg_count:
+    arg = args[arg_index]
```

Comment:
Can you use use `i` for the index? I think for a look like this it actually makes it more readable.